portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,42 +26,59 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    private List<String> messages;
+    
+    public class Name 
```

Comment:
I don't understand what this class does. It has no accessible fields or methods?